### DISCUSSION TITLE
Documentation for google-storage-bucket-encryption-customer-key not found

### DISCUSSION THREAD

**Discussion by takanakahiko** [aquasecurity/trivy#5957] [Documentation]
### Description

I can't find documentation for `google-storage-bucket-encryption-customer-key`.

When I run tfsec I get the following output.

````
Results #3-15 LOW Storage bucket encryption does not use a customer-managed key. (13 similar results)
──────────────────
  modules/common/functions.tf:2-6
   via projects/xxx/common.tf:1-12 (modules.common)
──────────────────
    2 resource "google_storage_bucket" "functions" {
    3 name = "functions_${google_project.project.name}"
    4 location = "US"
    5 uniform_bucket_level_access = true
    6 }
──────────────────
  Individual Causes
  - modules/common/functions.tf:1-8 (module.common)
──────────────────
          ID google-storage-bucket-encryption-customer-key
      Impact Using unmanaged keys does not allow for proper key management.
  Resolution Encrypt Cloud Storage buckets using customer-managed keys.

  More Information
  - https://aquasecurity.github.io/tfsec/v1.28.4/checks/google/storage/bucket-encryption-customer-key/
  - https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_bucket#encryption
````

The link <https://aquasecurity.github.io/tfsec/v1.28.4/checks/google/storage/bucket-encryption-customer-key/> in this is not a valid URL.

### Suggestions

Documentation for google-storage-bucket-encryption-customer-key should be created as <https://aquasecurity.github.io/tfsec/v1.28.4/checks/google/storage/bucket-encryption-customer-key/>.

**Comment by nikpivkin** [comment]
Hi @takanakahiko !

Now tfsec is part of Trivy, so you can use Trivy to scan IaC. https://github.com/aquasecurity/tfsec/discussions/1994

**Comment by simar7** [comment]
As @nikpivkin mentioned if you scan with Trivy you'll see it https://avd.aquasec.com/misconfig/google/storage/avd-gcp-0066/

**Comment by takanakahiko** [comment]
Thanks!
